clerk/tokens_options_test.go

```diff
@@ -9,36 +9,74 @@ import (
 
 func TestWithAuthorizedPartyNone(t *testing.T) {
 	opts := &verifyTokenOptions{}
-	WithAuthorizedParty()(opts)
+	err := WithAuthorizedParty()(opts)
 
-	assert.Len(t, opts.authorizedParties, 0)
+	if assert.NoError(t, err) {
+		assert.Len(t, opts.authorizedParties, 0)
+	}
```

Comment:
No need to use an if statement here, if there is an error the test will fail

Reply:
Hm, are you actually sure about that? https://github.com/stretchr/testify/blob/v1.7.1/assert/assertions.go#L1326

`Fail` only reports errors but doesn't abort the test: https://github.com/stretchr/testify/blob/083ff1c0449867d0d8d456483ee5fab8e0c0e1e6/assert/assertion_forward.go#L322

Reply:
Ah, you mean because it doesn't matter if `assert.Len` is called or not. In the past, I've used the NoError check to prevent any other tests to be executed against nil objects. I guess that doesn't matter here.

Reply:
Yes that's right, the test will not be aborted but eventually will be reported as failed. We don't care in this case if the rest of the code will be executed

Reply:
Done